google/auth/_default.py

```diff
@@ -69,14 +69,17 @@ def _warn_about_problematic_credentials(credentials):
         warnings.warn(_CLOUD_SDK_CREDENTIALS_WARNING)
 
 
-def _load_credentials_from_file(filename):
-    """Loads credentials from a file.
+def load_credentials_from_file(filename, scopes=None):
+    """Loads Google credentials from a file.
 
     The credentials file must be a service account key or stored authorized
     user credentials.
 
     Args:
         filename (str): The full path to the credentials file.
+        scopes (Sequence[str]): The list of scopes for the credentials. If
```

Comment:
Should there also be an `Optional`, as in `Optional[Sequence[str]]`? I can't remember exactly how the syntax works.